ethereum_history_api/oracles/src/noir/oracles/headerOracle/codec.ts

```diff
@@ -0,0 +1,30 @@
+import { ForeignCallOutput } from '@noir-lang/noir_js';
```

Comment:
How about naming files follow name conventions from files, i.e. this file would be 'encode.ts'?